testing/lxterminal-gtk3/APKBUILD

```diff
@@ -0,0 +1,41 @@
+# Contributor: Natanael Copa <ncopa@alpinelinux.org>
+# Contributor: Daniel YC Lin <dlin.tw@gmail.com>
+# Maintainer: Daniel YC Lin <dlin.tw@gmail.com>
+pkgname=lxterminal-gtk3
+_pkgname=lxterminal
+pkgver=0.3.0
+pkgrel=0
+pkgdesc="Desktop-independent VTE-based terminal emulator support gtk3 IME input"
+url="http://lxde.org/"
+arch="all"
+license="GPL2+"
+depends="vte3 gtk+3.0"
+makedepends="vte3-dev intltool gtk+3.0-dev"
+conflict="lxterminal"
```

Comment:
There’s no such variable in abuild. The correct approach is to add `!lxterminal` into `depends`.